app/modules/import-export-customization/assets/js/import/components/drop-zone.js

```diff
@@ -0,0 +1,303 @@
+		const validExtensions = getValidExtensions();
+
+		validExtensions.forEach( ( ext ) => {
+			acceptTypes.push( `.${ ext }` );
+		} );
+
+		return acceptTypes.join( ',' );
+	};
+
+	return (
+		<Box
+			className={ className }
+			onDragEnter={ handleDragEnter }
+			onDragLeave={ handleDragLeave }
+			onDragOver={ handleDragOver }
+			onDrop={ handleDrop }
+			data-testid="drop-zone"
+			{ ...props }
+		>
+			<Box
+				sx={ {
+					border: 2,
+					borderColor: isDragOver ? 'primary.main' : 'divider',
+					borderStyle: 'dashed',
+					backgroundColor: isDragOver ? 'action.hover' : 'background.default',
+					transition: 'all 0.2s ease-in-out',
+					cursor: isLoading ? 'not-allowed' : 'normal',
+					position: 'relative',
+					overflow: 'hidden',
+				} }
+			>
+				<Box
+					display="flex"
+					flexDirection="column"
+					alignItems="center"
+					justifyContent="center"
+					gap={ ( theme ) => theme.spacing( 1 ) }
+					p={ ( theme ) => theme.spacing( 6 ) }
+					minHeight={ ( theme ) => theme.spacing( 65 ) }
+				>
+					{ isLoading && (
+						<Box
+							position="absolute"
+							top={ 0 }
+							left={ 0 }
+							right={ 0 }
+							bottom={ 0 }
+							display="flex"
+							alignItems="center"
+							justifyContent="center"
+							bgcolor="action.hover"
+							zIndex={ 1 }
+							data-testid="loading-overlay"
+						>
+							<CircularProgress size={ 40 } data-testid="loading-spinner" />
+						</Box>
+					) }
+
+					<Box
+						sx={ {
+							opacity: isLoading ? 0.5 : 1,
+							transition: 'opacity 0.2s ease-in-out',
+						} }
+						data-testid="icon-container"
+					>
+						{ icon ? (
+							<Box
+								component="i"
+								className={ icon }
+								sx={ {
+									fontSize: '40px',
+									color: 'text.disabled',
+								} }
+								data-testid="custom-icon"
+							/>
+						) : (
+							<UploadIcon
+								sx={ {
+									fontSize: '40px',
+									color: 'text.disabled',
+								} }
+								data-testid="upload-icon"
+							/>
+						) }
+					</Box>
+
+					<Stack spacing={ 2 } alignItems="center" textAlign="center" data-testid="text-container">
+						<Typography
+							variant="body1"
+							component="div"
+							color="text.primary"
+							sx={ {
+								opacity: isLoading ? 0.5 : 1,
+								transition: 'opacity 0.2s ease-in-out',
+								fontSize: '16px',
+							} }
+							data-testid="main-text"
+						>
+							<Link
+								component="span"
+								onClick={ handleButtonClick }
+								sx={ {
+									cursor: 'pointer',
+								} }
+								data-testid="click-to-upload"
+							>
+								{ __( 'Click to upload', 'elementor' ) }
+							</Link>
+							{ ' ' + __( 'or drag and drop', 'elementor' ) }
+						</Typography>
+						<Typography
+							variant="body2"
+							color={ error ? 'error' : 'text.secondary' }
+							sx={ {
+								opacity: isLoading ? 0.5 : 1,
+								transition: 'opacity 0.2s ease-in-out',
+								fontSize: '14px',
```

Comment:
```suggestion
								fontSize: '14px',
```
Why is it needed? it should be applied by the variant.